experiments/dsl/src/test/scala/org/apache/gearpump/streaming/dsl/StreamAppSpec.scala

```diff
@@ -39,19 +40,19 @@ class StreamAppSpec  extends FlatSpec with Matchers with BeforeAndAfterAll {
 
 
   it should "be able to generate multiple new streams" in {
-    val app = new StreamApp("dsl", context)
-    app.fromCollection(List("A").iterator, 1)
-    app.fromCollection(List("B").iterator, 1)
+    val app = StreamApp("dsl", context)
+    app(List("A"), 1)
```

Comment:
It is not good to construct a app by a source, as one app can have multiple source.

For example:

val app = StreamApp("appName")
app.fromSource(Kafka).map.reduce.writeTo(sink)
app.from(anotherSource).writeTo(sink2)